R/summariseInternal.R

```diff
@@ -0,0 +1,207 @@
+
+summariseCountsInternal <- function(x, strata, counts) {
+  q <- c(
+    'dplyr::n()',
+    'dplyr::n_distinct(.data$person_id)',
+    'dplyr::n_distinct(.data$subject_id)'
+  ) |>
+    rlang::set_names(c("count_records", "count_subjecst", "count_subjects")) |>
+    purrr::keep(c("records", "person_id", "subject_id") %in% counts) |>
```

Comment:
is count_subjecst a typo?

Reply:
Nope, its is just that I allow to be person_id or subject_id but the resulting estimate will be always "number of subjects". At the end the second on dplyr::n_distinct(.data$subject_id) it is not used, so we can delete it if we want